ghost/i18n/locales/sk/newsletter.json

```
@@ -1,24 +1,24 @@
 {
-    "By {authors}": "",
-    "Comment": "",
-    "complimentary": "",
-    "Email": "",
-    "free": "",
-    "Keep reading": "",
-    "Less like this": "",
-    "Manage subscription": "",
-    "Member since": "",
-    "More like this": "",
-    "Name": "",
-    "paid": "",
-    "Subscription details": "",
-    "trialing": "",
-    "Unsubscribe": "",
-    "View in browser": "",
-    "You are receiving this because you are a <strong>%%{status}%% subscriber</strong> to {site}.": "",
-    "Your free trial ends on {date}, at which time you will be charged the regular price. You can always cancel before then.": "",
-    "Your subscription has been canceled and will expire on {date}. You can resume your subscription via your account settings.": "",
-    "Your subscription has expired.": "",
-    "Your subscription will expire on {date}.": "",
-    "Your subscription will renew on {date}.": ""
+    "By {authors}": "od",
+    "Comment": "Komentovať",
+    "complimentary": "bezplatný",
+    "Email": "Email",
+    "free": "bezplatným",
+    "Keep reading": "Prečítať celé",
+    "Less like this": "Chcem menej takýchto",
+    "Manage subscription": "Spravovať odber",
+    "Member since": "Členom od",
+    "More like this": "Chcem viac takýchto",
+    "Name": "Meno",
+    "paid": "plateným",
+    "Subscription details": "Podprobnosti odberu",
```

Should this be Podrobnosti odberu ?  

```suggestion
    "Subscription details": "Podrobnosti odberu",
```